src/app/components/auth/reset-password-confirm/reset-password-confirm.component.html

```diff
@@ -0,0 +1,71 @@
+               (focusin)="isNewPassword1Focused = true"
+               (focusout)="isNewPassword1Focused = authService.regUser['new_password1'] !== ''"
+               [(ngModel)]="authService.getUser['new_password1']" minlength="8" required>
+        <span class="form-icon form-icon-dark" (click)="authService.togglePasswordVisibility()">
+                    <i *ngIf="!authService.canShowPassword" class="fa fa-eye pointer"></i>
+                    <i *ngIf="authService.canShowPassword" class="fa fa-eye-slash pointer"></i>
+                </span>
+        <label for="new_password1" [class.active]="isNewPassword1Focused">New Password (min 8 characters) *</label>
+        <div class="wrn-msg text-highlight" *ngIf="newpassword1.invalid && (newpassword1.dirty || newpassword1.touched || resetconfirmform.submitted)">
+          <p *ngIf="newpassword1.errors.minlength">Password is less than 8 characters.</p>
+          <p *ngIf="newpassword1.errors.required">Password is required.</p>
+        </div>
+      </div>
+      <div class="input-field align-left">
+        <input type="{{authService.canShowConfirmPassword ? 'text' : 'password'}}" id="new_password2" class="dark-autofill" name="confirm_password" #newpassword2 = 'ngModel'
+               (focusin)="isNewPassword2Focused = true"
+               (focusout)="isNewPassword2Focused = authService.regUser['new_password_2'] !== ''"
+               [(ngModel)]="authService.getUser['new_password2']" minlength="8" required>
+        <span class="form-icon form-icon-dark" (click)="authService.toggleConfirmPasswordVisibility()">
+                    <i *ngIf="!authService.canShowConfirmPassword" class="fa fa-eye pointer"></i>
+                    <i *ngIf="authService.canShowConfirmPassword" class="fa fa-eye-slash pointer"></i>
+        </span>
+        <label for="new_password2" [class.active]="isNewPassword2Focused">Confirm new password *</label>
+        <div class="wrn-msg text-highlight" *ngIf="(newpassword2.invalid ) && (newpassword2.dirty || newpassword2.touched || resetconfirmform.submitted)">
+          <p *ngIf="newpassword2.errors.minlength">Password is less than 8 characters.</p>
+          <p *ngIf="newpassword2.errors.required">Password confirmation is required.</p>
+        </div>
+        <div class="wrn-msg text-highlight"
+             *ngIf="resetconfirmform.errors !== null && newpassword2.valid && (newpassword2.dirty || newpassword2.touched || resetconfirmform.submitted)">
+          <p *ngIf="resetconfirmform.errors['passwordMismatch']">Passwords do not match</p>
+        </div>
+      </div>
+      <div class="row">
+        <br>
+        <div class="col s6 align-left rm-gut">
+          <button class="waves-effect waves-dark btn ev-btn-dark w-300  grad-btn grad-btn-dark" type="submit" value="Submit">Submit</button>
+        </div>
+        <div class="col s6 align-right right rm-gut">
+          <a routerLink="/auth/login" class="light-link fg-pass">Back to Log In</a>
```

Comment:
`/auth/login` from `.ts` file. (Check throughout the code.)

Reply:
done